src/batch-client/index.ts

```diff
@@ -1704,7 +1706,10 @@ export class ZimbraBatchClient {
 			this.sessionId ||
 			(this.sessionHandler && this.sessionHandler.readSessionId()),
 		origin: this.origin,
-		userAgent: this.userAgent
+		userAgent: this.userAgent,
+		...(this.notifier.getSequenceNumber() && {
+			sessionSeq: this.notifier.getSequenceNumber()
```

Comment:
are we maintaining same seq number when request is fired through old zimbra-client?

Reply:
Ah, had missed that. The answer is, no we are not. I just checked that only `csrf` is added in old client, it is not adding `userAgent`, `sessionId`, `sessionSeq`, `accountName`, and `jwt`. Should we add them too?